File: c/addition/add.c
#include <stdio.h>

int Add(int a, int b)
{
	return a + b;
}

int main()
{
	int a = 0;
	int b = 0;
	int c = 0;

	printf("Choose two numbers to add.\n");
	scanf("%d%d", &a, &b);
	c = Add(a, b);
	printf("%d%s", c, "\n");
	return 0;
}

File: fortran/addition/add.f90
function add(a, b)
	integer :: add
	integer :: a
	integer :: b
	add = a + b
	return
end function add

program Addition
	implicit none
	integer :: a
	integer :: b
	integer :: add
	print*, "Which two numbers would you like to add?"
	read (*,*) a,b
	print*, add(a,b)
end program Addition
